zetaclient/chains/evm/observer/v2_inbound.go

```diff
@@ -181,6 +181,14 @@ func (ob *Observer) newDepositInboundVote(event *gatewayevm.GatewayEVMDeposited)
 		isCrossChainCall = true
 	}
 
+	// convert erc20Address to asset in foreign coin store to avoid checksum mismatch
```

Comment:
```suggestion
	// convert erc20Address to asset in foreign coin store to avoid checksum mismatch
	// convert erc20Address to asset in foreign coin store to avoid checksum mismatch
	// TODO: remove once the checksum conversion is fixed in the protocol
	// https://github.com/zeta-chain/node/issues/3274
```